diff -rua --unidirectional-new-file Debian7/etc/apache2/apache2.conf /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/apache2.conf
--- Debian7/etc/apache2/apache2.conf	2013-03-04 16:00:37.000000000 -0500
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/apache2.conf	2013-06-21 11:27:27.000000000 -0400
@@ -119,6 +119,7 @@
 #
 KeepAliveTimeout 5
 
+ServerName localhost
 ##
 ## Server-Pool Size Regulation (MPM specific)
 ## 
diff -rua --unidirectional-new-file Debian7/etc/apache2/mods-enabled/rewrite.load /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/mods-enabled/rewrite.load
--- Debian7/etc/apache2/mods-enabled/rewrite.load	1969-12-31 19:00:00.000000000 -0500
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/mods-enabled/rewrite.load	2013-06-21 11:27:27.000000000 -0400
@@ -0,0 +1 @@
+LoadModule rewrite_module /usr/lib/apache2/modules/mod_rewrite.so
diff -rua --unidirectional-new-file Debian7/etc/apache2/sites-available/default /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/sites-available/default
--- Debian7/etc/apache2/sites-available/default	2013-03-03 06:14:45.000000000 -0500
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/sites-available/default	2013-06-22 22:51:21.000000000 -0400
@@ -8,7 +8,8 @@
 	</Directory>
 	<Directory /var/www/>
 		Options Indexes FollowSymLinks MultiViews
-		AllowOverride None
+		#AllowOverride None
+		AllowOverride all
 		Order allow,deny
 		allow from all
 	</Directory>
diff -rua --unidirectional-new-file Debian7/etc/apache2/sites-enabled/000-default /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/sites-enabled/000-default
--- Debian7/etc/apache2/sites-enabled/000-default	2013-03-03 06:14:45.000000000 -0500
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/apache2/sites-enabled/000-default	2013-06-22 22:51:21.000000000 -0400
@@ -8,7 +8,8 @@
 	</Directory>
 	<Directory /var/www/>
 		Options Indexes FollowSymLinks MultiViews
-		AllowOverride None
+		#AllowOverride None
+		AllowOverride all
 		Order allow,deny
 		allow from all
 	</Directory>
diff -rua --unidirectional-new-file Debian7/etc/php5/apache2/conf.d/20-xdebug.ini /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/apache2/conf.d/20-xdebug.ini
--- Debian7/etc/php5/apache2/conf.d/20-xdebug.ini	2013-07-31 00:06:03.000000000 -0400
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/apache2/conf.d/20-xdebug.ini	2013-07-24 14:45:13.000000000 -0400
@@ -1 +1,5 @@
 zend_extension=/usr/lib/php5/20100525+lfs/xdebug.so
+xdebug.auto_trace=0
+xdebug.trace_output_dir=/tmp/traces
+xdebug.trace_output_name = trace.%c.%p
+#xdebug.trace_enable_trigger=1
diff -rua --unidirectional-new-file Debian7/etc/php5/cli/conf.d/20-xdebug.ini /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/cli/conf.d/20-xdebug.ini
--- Debian7/etc/php5/cli/conf.d/20-xdebug.ini	2013-07-31 00:06:03.000000000 -0400
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/cli/conf.d/20-xdebug.ini	2013-07-24 14:45:13.000000000 -0400
@@ -1 +1,5 @@
 zend_extension=/usr/lib/php5/20100525+lfs/xdebug.so
+xdebug.auto_trace=0
+xdebug.trace_output_dir=/tmp/traces
+xdebug.trace_output_name = trace.%c.%p
+#xdebug.trace_enable_trigger=1
diff -rua --unidirectional-new-file Debian7/etc/php5/conf.d/20-xdebug.ini /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/conf.d/20-xdebug.ini
--- Debian7/etc/php5/conf.d/20-xdebug.ini	2013-07-31 00:06:03.000000000 -0400
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/conf.d/20-xdebug.ini	2013-07-24 14:45:13.000000000 -0400
@@ -1 +1,5 @@
 zend_extension=/usr/lib/php5/20100525+lfs/xdebug.so
+xdebug.auto_trace=0
+xdebug.trace_output_dir=/tmp/traces
+xdebug.trace_output_name = trace.%c.%p
+#xdebug.trace_enable_trigger=1
diff -rua --unidirectional-new-file Debian7/etc/php5/mods-available/xdebug.ini /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/mods-available/xdebug.ini
--- Debian7/etc/php5/mods-available/xdebug.ini	2013-07-31 00:06:03.000000000 -0400
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/mods-available/xdebug.ini	2013-07-24 14:45:13.000000000 -0400
@@ -1 +1,5 @@
 zend_extension=/usr/lib/php5/20100525+lfs/xdebug.so
+xdebug.auto_trace=0
+xdebug.trace_output_dir=/tmp/traces
+xdebug.trace_output_name = trace.%c.%p
+#xdebug.trace_enable_trigger=1
diff -rua --unidirectional-new-file Debian7/etc/php5/mods-available/xdebug.ini~ /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/mods-available/xdebug.ini~
--- Debian7/etc/php5/mods-available/xdebug.ini~	1969-12-31 19:00:00.000000000 -0500
+++ /root/vulncorpus/framework/chroot_envs/Debian7/etc/php5/mods-available/xdebug.ini~	2013-06-21 11:27:27.000000000 -0400
@@ -0,0 +1,4 @@
+zend_extension=/usr/lib/php5/20100525+lfs/xdebug.so
+#xdebug.auto_trace=1
+xdebug.trace_output_dir=/tmp/traces
+xdebug.trace_output_name = trace.%c.%p.%d
